src/configure

```diff
@@ -628,7 +632,8 @@ function linux_configure_static {
   if $threaded_atlas; then pt=pt; else pt=""; fi
 
   if [ -z $ATLASLIBDIR ]; then # Note: it'll pick up the last one below.
-    for dir in /usr{,/local}/lib{64,}{,/atlas,/atlas-sse2,/atlas-sse3} \
```

Comment:
I don't think this is correct change -- LD_LIBRARY_PATH is a variable that should be used by linker. For linker during compilation, LIBRARY_PATH should be consulted.